CONTRIBUTING.md

```diff
@@ -1,10 +1,12 @@
 <!-- omit in toc -->
 # Contributing to Purple Bubble
 
-First off, thanks for taking the time to contribute! ❤️
+First off, thanks for taking the time to contribute! 💜
 
 All types of contributions are encouraged and valued. See the [Table of Contents](#table-of-contents) for different ways to help and details about how this project handles them. Please make sure to read the relevant section before making your contribution. It will make it a lot easier for us maintainers and smooth out the experience for all involved. The community looks forward to your contributions. 🎉
 
+For a list of things that we're currently working on (and that you can help with!) check out `ROADMAP.md`.
+
 > And if you like the project, but just don't have time to contribute, that's fine. There are other easy ways to support the project and show your appreciation, which we would also be very happy about:
 > - Star the project
 > - Post about it on social media
```

Comment:
Not really an avenue we would think about... not sure how I feel here.

Reply:
are you referring to the thing about social media?
if so i don't really like it either

Reply:
yes

Reply:
lets remove it.